docs/CHANGELOG.asciidoc

```diff
@@ -43,6 +44,62 @@ Review important information about the {kib} 8.x releases.
 * <<release-notes-8.0.0-alpha1>>
 
 --
+[[release-notes-8.8.2]]
+== {kib} 8.8.2
+
+coming::[8.8.2]
+
+Review the following information about the {kib} 8.8.2 release.
+
+[float]
+[[fixes-v8.8.2]]
+=== Bug Fixes
+
+APM::
+* Circuit breaker and performance improvements for service map {kibana-pull}159883[#159883]
+* Don't use service transaction metrics on transaction detail page {kibana-pull}159085[#159085]
```

Comment:
```suggestion
* Fixes the latency graph displaying all service transactions, rather than the selected one, on the transaction detail page {kibana-pull}159085[#159085]
```